scenario1/experiment_orchestrator.py

```diff
@@ -8,12 +8,12 @@
 
 NAMESPACE = "blis"
 
-def start_vllm_server_client(benchmark_config, exp_folder, mode, model, active_deadline_sec):
+def start_vllm_server_client(benchmark_config, exp_folder, mode, model):
     """Start vLLM server with config parameters"""
     vllm_params = benchmark_config['vllm']
 
     model_alias = model.split("/")[-1].replace(".", "_")
-    exp_results_path = f"/mnt/results/{model_alias}/{exp_folder}"
+    exp_results_path = f"/mnt/scenario1/results/{model_alias}/{exp_folder}"
```

Comment:
Should this be `scenario2`? Or instead of hardcoding it, make it configurable through config yaml?